journal-of-medical-internet-research.csl

```diff
@@ -218,6 +238,16 @@
       </if>
     </choose>
   </macro>
+  <macro name="webcite">
+    <choose>
+      <if type="webpage post-weblog" match="any">
+        <group delimiter=": " prefix=". ">
+          <text value="archived at" text-case="capitalize-first"/>
+          <text variable="note"/>
```

Comment:
I'd say just add it as part of the final edit. I don't see how we'd reasonably automate this, no. We could use the archive field, but that's not available for webpages, so of little use there.